client/lib/abtest/active-tests.js

```diff
@@ -77,4 +77,12 @@ module.exports = {
 		},
 		defaultVariation: 'add'
 	},
+	headstart: {
+		datestamp: '20160203',
+		variations: {
+			original: 20,
+			headstart: 80
```

Comment:
Hm. My testing wisdom is definitely lacking, but how would `notTested` be different from `original`?